pir/pircl.go

```diff
@@ -0,0 +1,7 @@
+//  +build pircl
+
+package pir
+
+import (
+	_ "github.com/privacylab/talek/pir/pircl"
```

Comment:
If I `go install github.com/privacylab/talek/cli/talekreplica` or similar, there's no link for it to know it should pull in those dependencies otherwise.  These files trigger a dependency on talek/pir to also include pir/pircl when the right build tags are specified.